### PR TITLE
[INLONG-793][Doc] Add docs for tubemq go sdk.

### PR DESCRIPTION
**Pull request by xuehuanran**
This PR for #793, add documents for tubemq go sdk in en/zh-cn two languages.

### CODE REVIEW

File: docs/sdk/tubemq-sdk/go.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: GO SDK
```
Comment (dockerzhang) [inline docs/sdk/tubemq-sdk/go.md:2]: GO SDK
->
Golang SDK

File: docs/sdk/tubemq-sdk/go.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+sidebar_position: 3
+---
+
+Similar to the C++/Python SDK, the GO SDK is also divided into two parts: Producer and Consumer. Here is an introduction to both.
```
Comment (dockerzhang) [inline docs/sdk/tubemq-sdk/go.md:6]: Producer and Consumer
->
`Producer` and `Consumer`

File: docs/sdk/tubemq-sdk/go.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+### Producer
+
+First, import some packages:
```
Comment (dockerzhang) [inline docs/sdk/tubemq-sdk/go.md:10]: import some packages
->
import necessary packages

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/tubemq-sdk/go.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+---
+title: GO SDK
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/tubemq-sdk/go.md:2]: ditto

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Pull request merged**